pkg/testing/tools/fleettools/fleet.go

```diff
@@ -144,3 +152,43 @@ func DefaultURL(ctx context.Context, client *kibana.Client) (string, error) {
 
 	return "", errors.New("unable to determine default fleet server URL")
 }
+
+// NewEnrollParams creates a new policy with monitoring logs and metrics,
+// an enrollment token and returns an EnrollParams with the information to enroll
+// an agent. If an error happens, it returns nil and a non-nil error.
+func NewEnrollParams(ctx context.Context, client *kibana.Client) (*EnrollParams, error) {
```

Comment:
How is this new test related to sending the agent version to components?
I don't mind more coverage but I fail to see the relation with the purpose of the PR